Formula/h/h26forge.rb

```diff
@@ -0,0 +1,19 @@
+class H26forge < Formula
+  desc "Tool for making syntactically valid but semantically spec-noncompliant videos"
+  homepage "https://github.com/h26forge/h26forge"
+  url "https://github.com/h26forge/h26forge/archive/refs/tags/latest-8dde7ec.tar.gz"
```

Comment:
we need a version tag rather than `latest-xxx`

Reply:
These are the only tags published by the project, unfortunately. Do I need to include an artificial version number for the formula?

Reply:
that means, we cannot include it into `core`, as there is no release

Reply:
There are releases, as you can see here: https://github.com/h26forge/h26forge/releases

It uses the commit as the version instead of a number.

Reply:
got it, raised a question about release scheme.

Reply:
H26Forge dev here 👋 

Let me know if the new release naming scheme works for y'all: https://github.com/h26forge/h26forge/releases/tag/latest-2024-07

Reply:
@wrv thanks for chipping in here, can you do 2024-07-06 rather than `latest-2024-07`?

Reply:
Done: https://github.com/h26forge/h26forge/releases/tag/2024-07-06 😄

Reply:
Thanks @wrv!! ❤️